packages/hardhat-core/test/internal/hardhat-network/provider/baseFeePerGas.ts

```diff
@@ -83,7 +88,7 @@ describe("Block's baseFeePerGas", function () {
             });
           });
         } else {
-          describe("When forking from a remote network the next block must have the right value", function () {
+          describe("When forking from a remote network the forked block must have the right value", function () {
```

Comment:
```suggestion
          describe("When forking from a remote network the next block must have the right value", function () {
```

I think this is outdated?